webservices/test_api.py

```diff
@@ -177,19 +177,84 @@ def test_name_endpoint_returns_unique_candidates_and_committees(self):
     #     self.assertEquals(len(response['results']), 1)
 
 #Committee
+    # new
+    def test_committee_search_fields(self):
+        # example with committee
+        response = self._response('/committee?committee_id=C00048587&year=*')
+        result = response['results'][0]
+        # main fields
+        # original registration date doesn't make sense in this example, need to look into this more
+        self.assertEqual(result['original_registration_date'], '1982-12-31 00:00:00')
```

Comment:
Not a blocker for merging... But this test can be refactored to use the assertResultsEqual helper method.